### ISSUE TITLE
Project Proposal Review

### ISSUE THREAD

**Issue by enagola** [COGS118A/Group040-Sp22#1]
Rubric items are out of 4. 3/4 is still full credit, but it means you need to need to improve this item.

Title & Abstract (3/4)

The idea is there but you need to be more specific about the issue itself and the variables surrounding it.

Problem Statement (2/4)

You have identified what you are trying to solve but you are once again too broad in defining what needs to be solved.

Data (4/4)

Proposed Solution (2/4)

You suggested that you are going to use certain libraries but you should go more into detail of how you plan to do so. What your strategy is going to include and a little more research on the libraries that you plan to use and how you plan to use them.

Metrics (3/4)

Could be useful to give concrete examples of how this metric would work to give more of a clear understanding of how you will measure your model.

Ethics & Privacy (4/4)

Team Expectations (2/4)

Expectations are too broad. Need to have a concrete set of expectations as this will help the team work together and progress in a clear manner.

Timeline (4/4)

Comments:

Overall, it is an interesting idea but you need to be more specific about what you plan to do. The goals, expectations, and ideas for the solution need to be more specific as it will help you progress in the project more concretely and easily.

Basically, be more specific in everything
